pkg/standalone/standalone.go

```diff
@@ -59,13 +59,20 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 // TODO: replace this with clientcmd
```

Comment:
If you're feeling up to it we would like to collapse down onto clientcmd, but I don't think it's a hard requirement at this point.